pkg/lwip/patches/0002-lowpan6-Use-arch-independent-printf-formatters.patch

```diff
@@ -0,0 +1,51 @@
+From e02d126b07388c5ff49a6cd25cef14214157b52f Mon Sep 17 00:00:00 2001
+From: Erik Ekman <erik@kryo.se>
+Date: Sat, 13 Feb 2021 21:14:29 +0100
+Subject: [PATCH] lowpan6: Use arch-independent printf formatters
```

Comment:
Did you provide this patch upstream btw?

Reply:
Yes, I commited it there first: https://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=e02d126b07388c5ff49a6cd25cef14214157b52f